pinot-query-runtime/src/main/java/org/apache/pinot/query/service/server/QueryServer.java

```diff
@@ -83,24 +83,23 @@ public class QueryServer extends PinotQueryWorkerGrpc.PinotQueryWorkerImplBase {
 
   @VisibleForTesting
   public QueryServer(int port, QueryRunner queryRunner, @Nullable TlsConfig tlsConfig) {
-    this(port, queryRunner, tlsConfig, ServerMetrics.get(), new PinotConfiguration());
+    this(port, queryRunner, tlsConfig, new PinotConfiguration());
   }
 
-  public QueryServer(int port, QueryRunner queryRunner, @Nullable TlsConfig tlsConfig,
-      ServerMetrics serverMetrics, PinotConfiguration config) {
+  public QueryServer(int port, QueryRunner queryRunner, @Nullable TlsConfig tlsConfig, PinotConfiguration config) {
     _port = port;
     _queryRunner = queryRunner;
     _tlsConfig = tlsConfig;
 
-    ExecutorService baseExecutor = Executors.newCachedThreadPool(
-        new NamedThreadFactory("query_submission_executor_on_" + _port + "_port"));
+    ExecutorService baseExecutor =
+        Executors.newCachedThreadPool(new NamedThreadFactory("query_submission_executor_on_" + _port + "_port"));
```

Comment:
nit: there are some style changes applied here and in QueryRunner that seem to point that your IDE is automatically applying refactors that are not required by the codestyle

Reply:
Let me revert the unrelated changes